webhooks/cluster.go

```diff
@@ -182,34 +182,34 @@ func (webhook *Cluster) validateTopology(ctx context.Context, old, new *clusterv
 		)
 	}
 
-	// MachineDeployment names must be unique.
-	if new.Spec.Topology.Workers != nil {
-		names := sets.String{}
-		for _, md := range new.Spec.Topology.Workers.MachineDeployments {
-			if names.Has(md.Name) {
-				allErrs = append(allErrs,
-					field.Invalid(
-						field.NewPath("spec", "topology", "workers", "machineDeployments"),
-						md,
-						fmt.Sprintf("MachineDeployment names should be unique. MachineDeployment with name %q is defined more than once.", md.Name),
-					),
-				)
-			}
-			names.Insert(md.Name)
-		}
+	// clusterClass must exist.
+	clusterClass := &clusterv1.ClusterClass{}
+	// Check to see if the ClusterClass referenced in the Cluster currently exists.
+	if err := webhook.Client.Get(ctx, client.ObjectKey{Namespace: new.Namespace, Name: new.Spec.Topology.Class}, clusterClass); err != nil {
+		allErrs = append(
+			allErrs, field.Invalid(
```

Comment:
```suggestion
		allErrs = append(allErrs
		    , field.Invalid(
```
nit

Reply:
I think it's slightly better as l.190++ is then essentially the equivalent to just a struct literal